provd/debian/postinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
```

Comment:
You should use the debhelper tags for those, please have a look at other projects with the debian/ directory and systemd services.

Here, you are not doing anything special outside of the systemd services itself, so it should be generated by dh_systemd automatically for you.